metis-normalization/src/test/java/eu/europeana/normalization/dates/extraction/extractors/MonthNameDateExtractorTest.java

```diff
@@ -0,0 +1,161 @@
+package eu.europeana.normalization.dates.extraction.extractors;
+
+import static org.junit.jupiter.params.provider.Arguments.of;
+
+import eu.europeana.normalization.dates.DateNormalizationExtractorMatchId;
+import eu.europeana.normalization.dates.DateNormalizationResult;
+import java.util.stream.Stream;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+class MonthNameDateExtractorTest implements DateExtractorTest {
+
+  private static final MonthNameDateExtractor PATTERN_MONTH_NAME_DATE_EXTRACTOR = new MonthNameDateExtractor();
+
+  @ParameterizedTest
+  @MethodSource
+  void extractDayMonthYear(String input, String expected) {
+    assertExtract(input, expected);
+  }
+
+  @ParameterizedTest
+  @MethodSource
+  void extractMonthDayYear(String input, String expected) {
+    assertExtract(input, expected);
+  }
+
+  @ParameterizedTest
+  @MethodSource
+  void extractMonthYear(String input, String expected) {
+    assertExtract(input, expected);
+  }
+
+  void assertExtract(String input, String expected) {
+    final DateNormalizationResult dateNormalizationResult = PATTERN_MONTH_NAME_DATE_EXTRACTOR.extractDateProperty(input);
+    assertDateNormalizationResult(dateNormalizationResult, expected, DateNormalizationExtractorMatchId.MONTH_NAME);
+  }
+
+  private static Stream<Arguments> extractDayMonthYear() {
+    return Stream.of(
+        //DAY-MONTH-YEAR
```

Comment:
Month names in this languages were not accepted don't know if the structure `EuropeanLanguage` supports it.
```
of("Marraskuu 01 1989", "1989-11-01"), //Finnish
of("Listopad 01 1989", "1989-11-01"), //Polish
of("Novembre 01 1989", "1989-11-01"), //Italian
of("Noviembre 01 1989", "1989-11-01"), //Spanish
```

Reply:
Indeed! I've fixed it now.